sql/codegen_xgboost_test.go

```diff
@@ -321,13 +326,11 @@ LABEL e INTO model_table;
 `
 	pr, e := parser.Parse(trainClause)
 	a.NoError(e)
-	fts, e := verify(pr, testDB)
-	a.NoError(e)
-	filler, e := newXGBoostFiller(pr, nil, fts, testDB)
+	filler, e := newXGBoostFiller(pr, nil, testDB)
 	a.NoError(e)
 
 	a.True(filler.IsTrain)
-	stdSlct := removeLastSemicolon(strings.Replace(filler.StandardSelect, "\n", " ", -1))
+	stdSlct := trimTailOf(strings.Replace(filler.StandardSelect, "\n", " ", -1), ';')
```

Comment:
Can use the standar function [strings.TrimRight](https://golang.org/pkg/strings/#TrimRight)

Reply:
Thanks for your reminder. I use [strings.TrimSuffix](https://golang.org/pkg/strings/#TrimSuffix) to replace `trimTailOf`.